data_prototype/containers.py

```diff
@@ -34,6 +34,7 @@ class Desc:
 class DataContainer(Protocol):
     def query(
         self,
+        /,
```

Comment:
not exactly sure why this is here... `self` is never passed as a kwarg anyway and this has no effect on the other arguments.

Reply:
🤦🏻 coding while sleepy and watching TV.  This should have been at the end.